packages/logger/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@loopback/logger",
+  "version": "1.0.0",
+  "description": "A LoopBack component for logger support.",
+  "scripts": {
+    "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "node ../../bin/compile-package es2017",
+    "build:lib6": "node ../../bin/compile-package es2015",
+    "clean": "rm -rf loopback-logger*.tgz lib* package",
+    "integration": "mocha --opts ../../test/mocha.opts 'test/integration/**/*.ts'",
+    "prepare": "npm run build",
+    "pretest": "npm run build",
```

Comment:
This should be `npm run build:current` instead. I think we may have changed npm scripts since you started the work on this logger component. Please check e.g. [packages/core/package.json](https://github.com/strongloop/loopback-next/blob/021050065007937b8935f086fd4a4b230eb9396d/packages/core/package.json) for the correct content.